configs/yolo/yolov3_d53_320_273e_coco_ipu.py

```diff
@@ -0,0 +1,98 @@
+_base_ = './yolov3_d53_mstrain-608_273e_coco.py'
+# dataset settings
+IM_SIZE = 320
+custom_hooks = [dict(type='NumClassCheckHook'), dict(type='BNToFP32')]
+img_norm_cfg = dict(mean=[0, 0, 0], std=[255., 255., 255.], to_rgb=True)
+train_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(type='LoadAnnotations', with_bbox=True),
+    dict(
+        type='Expand',
+        mean=img_norm_cfg['mean'],
+        to_rgb=img_norm_cfg['to_rgb'],
+        ratio_range=(1, 2)),
+    dict(
+        type='MinIoURandomCrop',
+        min_ious=(0.4, 0.5, 0.6, 0.7, 0.8, 0.9),
+        min_crop_size=0.3),
+    dict(type='Resize', img_scale=(IM_SIZE, IM_SIZE), keep_ratio=True),
+    dict(type='RandomFlip', flip_ratio=0.5),
+    dict(type='BGR2RGB'),
+    dict(type='Pad', size=(IM_SIZE, IM_SIZE)),
+    dict(
+        type='IPUFormatBundle',
+        img_to_float=False,
+        pad_dic=dict(
+            gt_bboxes=dict(dim=0, shape=96),
+            gt_labels=dict(dim=0, shape=96),
+            gt_bboxes_ignore=dict(dim=0, shape=20))),
+    dict(
+        type='IPUCollect',
+        keys=['img', 'gt_bboxes', 'gt_labels'],
+        meta_on=True),
+    dict(
+        type='GetTargetsOutsideForYolo',
+        featmap_sizes=[IM_SIZE // 32, IM_SIZE // 16, IM_SIZE // 8])
+]
+test_pipeline = [
+    dict(type='LoadImageFromFile'),
```

Comment:
Seems to be able to read as rgb at the start without `BGR2RGB`. `dict(type='LoadImageFromFile',channel_order='rgb')`